docs/_template.html

```diff
@@ -103,8 +103,26 @@
   else{
     document.getElementById(e.target.id+'-wrapper').classList.add('menu-open');
   }
-})
-});
+      })
```

Comment:
It might be a good time to put all this JS in a separate script.
It is becoming lengthy.